openapi-v1.yaml

```diff
@@ -6041,3 +6148,180 @@ paths:
           description: error response
           schema:
             $ref: "#/definitions/Error"
+
+  /groups/{namespace}:
+    parameters:
+      - name: namespace
+        type: string
+        in: path
+        required: true
+        description: The namespace to operate on
+    get:
+      description: Returns the attributes of one page of top level groups in namespace.
+      operationId: listTopLevelGroups
+      tags:
+        - groups
+      parameters:
+        - name: page
+          in: query
+          description: pagination offset
+          type: integer
+          required: false
+        - name: per_page
+          in: query
+          description: pagination limit
+          type: integer
+          required: false
+      responses:
+        200:
+          description: the group contents
+          schema:
+            $ref: '#/definitions/GroupListingTopLevel'
+        default:
+          description: error response
+          schema:
+            $ref: "#/definitions/Error"
+    post:
```

Comment:
I would expect post to be under `/groups/{namespace}/{name}` (please also see comment regarding `GroupCreate`)

Reply:
:+1: as discussed previously